### DISCUSSION TITLE
Proposal for Flagger logo

### DISCUSSION THREAD

**Discussion by bia** [fluxcd/flux2#653] [Proposals] [answered]
As discussed in today's meeting, we would like to create a CNCF logo for flagger, designed as if in the same family as the flux logo.
![210106flux-logo@2x](https://user-images.githubusercontent.com/92439/103758684-ce8e5580-5012-11eb-90aa-94b661542d8f.png)

Here's an initial sketch, informed by the diagram below
![210106flagger-logo@2x](https://user-images.githubusercontent.com/92439/103758473-8a02ba00-5012-11eb-91ac-0c6c6dbdcd88.png)

![](https://raw.githubusercontent.com/fluxcd/flagger/main/docs/diagrams/flagger-canary-overview.png)

**Comment by dholbach** [comment]
I like it... How did we pick the green :green_heart: ?

**Reply by bia** [reply]
Similar saturation and lightness, just with the hue shifted to green instead of blue

**Comment by bia** [comment]
Here are some further iterations from our discussion.

Overlapping rhombi instead of 3 rhombi so that it doesn't look like one version better than flux
![210106flagger-logo2@2x](https://user-images.githubusercontent.com/92439/103761358-d64ff900-5016-11eb-9ce6-22535b843a17.png)

@squaremo's idea of breaking down multiple arrows vertically
![210106flagger-logo3@2x](https://user-images.githubusercontent.com/92439/103761731-6c841f00-5017-11eb-8e97-1b929ef7f046.png)

@squaremo's idea of breaking down multiple arrows horizontally
I like this. Would have to play around with the arrow style a little more.
![210106flagger-logo4@2x](https://user-images.githubusercontent.com/92439/103762259-47dc7700-5018-11eb-8030-69e936e3f945.png)
![210106flagger-logo6@2x](https://user-images.githubusercontent.com/92439/103762881-52e3d700-5019-11eb-8b31-4d809f44f9f9.png)

**Reply by kingdonb** [reply]
My favorite version of the logo in the discussion so far is the first (two rhombi, overlapping vertically arranged) – I also like the look of the signal flags @squaremo described, but don't know how they will look when put together. The arrow makes sense to me, it looks quite like a symbol representing the canary release being promoted to primary.

Also I am partial to the green color used here myself, but seeing discussion below, those colors look fine too. (I think I'd suggest, don't take any color advice from me, as I personally don't have any sense of matching or clashing colors.)

**Comment by bia** [comment]
@hiddeco noticed that the green ![](https://placehold.it/15/228B22/000000?text=+) _#228B22_ above does not really work with flux's blue ![](https://placehold.it/15/326CE5/000000?text=+) _#326CE5_

We both are now leaning towards a green more like ![](https://placehold.it/15/7BB09F/000000?text=+) _#7BB09F_ : 

![210112flagger-logo3@2x](https://user-images.githubusercontent.com/92439/104335084-5b458180-54f3-11eb-948b-01a1a7a10dc6.png)

**Comment by csand-msft** [comment]
Can we see the family of Flux logos to get wholistic view?

**Comment by dholbach** [comment]
https://branding.cncf.io/projects/flux/

**Comment by bia** [comment]
Here are some flag flux logos from our discussion yesterday (I'm not a fan, but maybe you have suggestions from improvement) : 
![210115flagger-logo@2x](https://user-images.githubusercontent.com/92439/104764560-d7e09600-5767-11eb-9474-dcaee1afce20.png) ![210115flagger-logo2@2x](https://user-images.githubusercontent.com/92439/104764563-d8792c80-5767-11eb-8cb1-f993b7743924.png)

Then iterations on the diagonal arrows : 
![210115flagger-logo3@2x](https://user-images.githubusercontent.com/92439/104764723-14ac8d00-5768-11eb-9958-8514b9d19b2c.png) ![210115flagger-logo4@2x](https://user-images.githubusercontent.com/92439/104764725-15452380-5768-11eb-9c68-b52563b2dc5d.png)

Which brought me to these. What do we think : 
![210115flagger-logo5@2x](https://user-images.githubusercontent.com/92439/104764800-2b52e400-5768-11eb-8001-bc049cc53b0f.png) ![210115flagger-logo6@2x](https://user-images.githubusercontent.com/92439/104764802-2beb7a80-5768-11eb-86f5-c4b5713e2c6a.png)

**Reply by phillebaba** [reply]
Its cool seeing the progress of the logo.

It might be a bit corny but I really like the pointy golf flag in the logo. If we are going for brand recognition however it might be better to go with the green with three diagonal arrows.

**Reply by squaremo** [reply]
The name "Flagger" refers to the job of directing traffic around roadworks or a construction site; the flag suggested is more a signalling flag (or sign) than a golf flag. Signalling flags are typically squarish, with two colours divided on the diagonal, which could map nicely to the lozenge shapes.

![image](https://user-images.githubusercontent.com/62033/105021291-49f2fc80-5a40-11eb-87f6-c2eeb206d4cb.png)

**Reply by bia** [reply]
Nice! What do we think of this?
![210125flagger-logo@2x](https://user-images.githubusercontent.com/92439/105737892-bd13db80-5f36-11eb-85a1-b09efbd8247e.png)

**Reply by kingdonb** [reply]
💯

**Reply by stealthybox** [reply]
Ahh! That's really cool

**Reply by stefanprodan** [reply]
Looks great! Thanks Bianca

**Comment by bia** [comment]
Here is a logo pack with just the icons, full logo coming soon!
[flagger-icon-logo-pack.zip](https://github.com/fluxcd/flux2/files/5972061/flagger-icon-logo-pack.zip)

**Reply by stefanprodan** [reply]
Thank you Bianca 🥇

**Comment by bia** [comment, marked as answer]
Here is the full logo pack, including two logos with the name.
[flagger-logo-pack.zip](https://github.com/fluxcd/flux2/files/5991356/flagger-logo-pack.zip)


![flagger-stacked-color@2x](https://user-images.githubusercontent.com/92439/108114652-5dbe6c80-7099-11eb-930b-c5bc13665522.png) ![flagger-vertical-color@2x](https://user-images.githubusercontent.com/92439/108114726-79297780-7099-11eb-9c20-046d13423414.png)
